### PR TITLE
Rewritten usecase section on YASGUI

### PR DESCRIPTION
**Pull request by LaurensRietveld**
related to issue #92

### CODE REVIEW

File: Overview.html [hunk 1]
```diff
@@ -1698,7 +1698,12 @@ <h3 id="s8_6">9.6 Sampling Large RDF Graphs</h3>
...
+    <p>YASGUI [<a href="#YASGUI">YASGUI</a>] is a query builder for SPARQL with a strong focus on usability. 
+    To assist the user in formulating queries, YASGUI provides prefix autocompletion autocompletion using prefix.cc [<a href="#prefixcc">Prefix.cc</a>], endpoint autocompletion using the CKAN-based datahub.io [<a href="#Datahub">Datahub</a>], and property/class autocompletion using the LOV [<a href="#LOV">LOV</a>] API 
+    However, none of these autocompletions are based on the users dataset where the query is written for.
+    What users need are autocompletions based on the target dataset, in a fast and scalable manner. 
+    However, querying the complete dataset for this information via SPARQL, can be expensive and slow. Additionally, the number of suggestions can be large. 
+    The Bio2RDF metrics enables fetching such autocompletions in a fast manner, supporting meaningful ranking of the suggestions as well (e.g. by how often a particular class co-occurs with the current predicate).</p>
```
Comment (micheldumontier) [inline Overview.html]: replace Bio2RDF with "dataset"

**Comment by LaurensRietveld** [comment]
@micheldumontier : (force)updated PR per your comment

File: Overview.html [hunk 2]
```diff
@@ -1468,7 +1468,7 @@ <h3 id="s6_6">6.6 Statistics</h3>
     <p>To specify the <strong>properties and the number of unique literals</strong> in the dataset:</p>
 
     <pre>
-:rdfdataset
+:rdfdatasetreplace Bio2RDF with "datas
```
Comment (micheldumontier) [inline Overview.html]: ooops!

Comment (LaurensRietveld) [inline Overview.html]: pfff, oops indeed! fixed in most recent PR commit

**Comment by micheldumontier** [comment]
good. many thanks!

**Pull request merged**